api/src/lib/middleware/loggingRecoveryMiddleware.ts

```diff
@@ -2,7 +2,7 @@ import {createRecoveryMiddleware} from '@faaskit/core'
 import {logger} from '@lib/logging'
 
 const sendError = async (error, _event, _context) => {
-  logger.error(error)
+  logger.info(error)
```

Comment:
I switched this to info until I can better investigate that error isn't exiting prematurely.